proposal/metrics/enhancer.go

```diff
@@ -1,40 +1,95 @@
 package metrics
 
 import (
+	"sync"
+
 	v3 "github.com/mysteriumnetwork/discovery/proposal/v3"
 	"github.com/mysteriumnetwork/discovery/quality"
+	"github.com/mysteriumnetwork/discovery/quality/oracleapi"
 	"github.com/rs/zerolog/log"
 )
 
-type Enhancer struct {
-	qualityService *quality.Service
+type OracleResponses struct {
+	QualityResponse   *oracleapi.ProposalQualityResponse
+	LatencyResponse   *oracleapi.LatencyResponse
+	BandwitdhResponse *oracleapi.BandwidthResponse
+	SessionResponse   *oracleapi.SessionsResponse
 }
 
-func NewEnhancer(qualityService *quality.Service) *Enhancer {
-	return &Enhancer{qualityService: qualityService}
+func (or *OracleResponses) Load(qualityService *quality.Service, fromCountry string) {
```

Comment:
Can we be sure that load is called only from one goroutine? Shouldn't assignment of shared state be protected with mutex?